src/current/_includes/v24.1/known-limitations/physical-cluster-replication.md

```diff
@@ -1,7 +1,5 @@
-- Physical cluster replication is supported only on CockroachDB {{ site.data.products.core }} in new clusters on v23.2 or above. Physical Cluster Replication cannot be enabled on clusters that have been upgraded from a previous version of CockroachDB.
+
 - Read queries are not supported on the standby cluster before [cutover]({% link {{ page.version.version }}/cutover-replication.md %}).
-- The primary and standby cluster **cannot have different [region topology]({% link {{ page.version.version }}/topology-patterns.md %})**. For example, replicating a multi-region primary cluster to a single-region standby cluster is not supported. Mismatching regions between a multi-region primary and standby cluster is also not supported.
-- Cutting back to the primary cluster after a cutover is a manual process. Refer to [Cut back to the primary cluster]({% link {{ page.version.version }}/cutover-replication.md %}#cut-back-to-the-primary-cluster). In addition, after cutover, to continue using physical cluster replication, you must configure it again.
+- The primary and standby cluster **cannot have different [region topology]({% link {{ page.version.version }}/topology-patterns.md %})**. For example, replicating a multi-region primary cluster to a single-region standby cluster is not supported. Mismatching regions between a multi-region primary and standby cluster is also not supported. [#119934](https://github.com/cockroachdb/cockroach/issues/119934)
 - Before cutover to the standby, the standby cluster does not support running [backups]({% link {{ page.version.version }}/backup-and-restore-overview.md %}) or [changefeeds]({% link {{ page.version.version }}/change-data-capture-overview.md %}).
-- After a cutover, there is no mechanism to stop applications from connecting to the original primary cluster. It is necessary to redirect application traffic manually, such as by using a network load balancer or adjusting DNS records.
```

Comment:
Is this a limitation? It is in a warning box near the top of the cutover page.

Reply:
yeah, i think it's a known limitation as well.

Reply:
Yes, it is tricky, without knowing the planned work... we have these definitions for what is a known limitation vs. an unsupported feature:

> - Known limitations describe functionality that is suboptimal or not working.
> - 
> - Unsupported features are absent from the product. They may be added in the future (e.g., an aspect of a preview feature that is not yet implemented) unless they are absent by design.

This could fall into either, I think, depending on if we're going to resolve this in some way. Still known limitation?

Reply:
we plan to resolve this for 24.2, so i guess it is unsupported.

Reply:
As per a Slack conversation, will keep this as a KL and add in the related GitHub issue.